dream_layer_backend/txt2img_server.py

```
@@ -59,28 +103,96 @@ def handle_txt2img():
+            # Start timing 
+            start = time.perf_counter()
+            
             # Send to ComfyUI server
             comfy_response = send_to_comfyui(workflow)
             
+            # Stop timing
+            elapsed = time.perf_counter() - start
+
+            # Determine number of images generated and time per image
+            images_generated = len(comfy_response.get("all_images", []))
+            time_per_image = None if images_generated == 0 else total_time / images_generated
```

Ruff (0.12.2)

Undefined name `total_time`

(F821)